extensions/Push/lib/Connector/Phabricator.pm

```diff
@@ -0,0 +1,81 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# # License, v. 2.0. If a copy of the MPL was not distributed with this
+# # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# #
+# # This Source Code Form is "Incompatible With Secondary Licenses", as
+# # defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::Extension::Push::Connector::Phabricator;
+
+use 5.10.1;
+use strict;
+use warnings;
+
+use base 'Bugzilla::Extension::Push::Connector::Base';
+
+use Bugzilla::Bug;
+use Bugzilla::Constants;
+use Bugzilla::Extension::Push::Constants;
+use Bugzilla::Extension::Push::Util;
+use Bugzilla::User;
+
+use constant PHAB_CONTENT_TYPE => 'text/x-phabricator-request';
+use constant SECURITY_GROUP_KEY => '?'; # TODO:  Need to figure out what key this is
```

Comment:
this will be part of phase 2 or 3 of the connector development. But the end goal would be we would look at the current list of groups of a bug (after change) and the list of groups we are currently syncing from BMO to Phab, and make the new custom policy of the revision to be the intersection of those two group lists.

The synced groups list will come from Bugzilla->params->{phabricator_sync_groups}

So we do not actuallt need the SECURITY_GROUP_KEY constant.